client/lib/wporg/index.js

```diff
@@ -106,12 +108,53 @@ module.exports = {
 	 * @returns {?Promise} Promise  that is returned if no callback parameter is passed
 	 */
 	fetchThemeInformation: function( themeId, callback ) {
-		const url = 'https://api.wordpress.org/themes/info/1.1/';
 		const query = { action: 'theme_information', 'request[slug]': themeId };
 		// if callback is provided, behave traditionally
 		if ( 'function' === typeof callback ) {
 			return superagent
-				.get( url )
+				.get( _WPORG_THEMES_ENDPOINT )
+				.set( 'Accept', 'application/json' )
+				.query( query )
+				.end( ( err, { body } ) => {
+					callback( err, body );
+				} );
+		}
+
+		// otherwise, return a Promise
```

Comment:
Decided to go with a YAGNI approach and axed the callback option altogether. Discovered also that `superagent`, if called without `.end()`, [returns a promise](http://visionmedia.github.io/superagent/#promise-and-generator-support).